test/colorado_lottery_test.rb

```diff
@@ -0,0 +1,179 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require 'pry'
+require "./lib/game"
+require "./lib/contestant"
+require "./lib/colorado_lottery"
+
+class ColoradoLotteryTest < Minitest::Test
+
+  def setup
+    @lottery = ColoradoLottery.new
+    @pick_4 = Game.new('Pick 4', 2)
+    @mega_millions = Game.new('Mega Millions', 5, true)
+    @cash_5 = Game.new('Cash 5', 1)
+    @alexander = Contestant.new({
+                 first_name: 'Alexander',
+                 last_name: 'Aigades',
+                 age: 28,
+                 state_of_residence: 'CO',
+                 spending_money: 10})
+    @benjamin = Contestant.new({
+                 first_name: 'Benjamin',
+                 last_name: 'Franklin',
+                 age: 17,
+                 state_of_residence: 'PA',
+                 spending_money: 100})
+    @frederick = Contestant.new({
+                  first_name:  'Frederick',
+                  last_name: 'Douglas',
+                  age: 55,
+                  state_of_residence: 'NY',
+                  spending_money: 20})
+    @winston = Contestant.new({
+                  first_name: 'Winston',
+                  last_name: 'Churchill',
+                  age: 18,
+                  state_of_residence: 'CO',
+                  spending_money: 5})
+    @grace = Contestant.new({
+                  first_name: 'Grace',
+                  last_name: 'Hopper',
+                  age: 20,
+                  state_of_residence: 'CO',
+                  spending_money: 20})
+    @alexander.add_game_interest('Pick 4')
+    @alexander.add_game_interest('Mega Millions')
+    @frederick.add_game_interest('Mega Millions')
+    @winston.add_game_interest('Cash 5')
+    @winston.add_game_interest('Mega Millions')
+    @benjamin.add_game_interest('Mega Millions')
```

Comment:
Love seeing all of this test setup in a setup method. An important time saver, and makes the code DRY.